### PR TITLE
Lmaoo127

### PR DESCRIPTION
**Pull request by LewisWelch94**
Can not automatically merge for some reason.

### CODE REVIEW

File: Ticket/createComment.php [hunk 1]
```diff
@@ -3,4 +3,4 @@
 <div ID="manychars" class="alert alert-warning " role="alert" hidden>Text is to long inside text field!</div>
 <script type="text/javascript" src="../Script/commentController.js"></script>
 <textarea id="createComment" class=createComment name="editordata"></textarea>
-<button onclick="saveComment('.createComment')">Save Comment</button> 
\ No newline at end of file
+<button id="saveComment" onclick="saveComment('.createComment', 0)">Save Comment</button>
```
Comment (OnikNoor97) [inline Ticket/createComment.php:6]: Get rid of , 0

File: Script/peopleController.js [hunk 2]
```diff
@@ -27,23 +27,26 @@ function loadAssignee()
...
+    $("#assigneeUserId").html(res[0].userId);
+  })
 }
 
 function loadReporter()
 {
   var ticketId = new URL(window.location.href).searchParams.get("ticketId");
   loadReporterFromServer(ticketId)
   .then(response => 
+
```
Comment (OnikNoor97) [inline Script/peopleController.js:44]: Get rid of new line here

File: Css/ticketPage.css [hunk 3]
```diff
@@ -71,12 +71,17 @@
 
 #ticketCreate {
 
-	text-align: centre;
+	text-align: left;
 	width: 1090px;
 	margin-top: 100px;
 	margin-left: auto;
 	margin-right: auto;
```
Comment (OnikNoor97) [inline Css/ticketPage.css:78]: Use clockwise notation for this

File: Ticket/createComment.php (hunk 1 above)
Comment (LewisWelch94) [reply Ticket/createComment.php]: done

**Pull request merged**